sdk/openai/azure-ai-openai/CHANGELOG.md

```diff
@@ -17,14 +17,36 @@
 
 **Chat**
 
+- [AOAI] Added new class `OnYourDataContextProperty` to represent the context property in the On Your Data deployment.
```

Comment:
[Here](https://dev.azure.com/project-argos/Sorento/_wiki/wikis/Sorento.wiki/2000/2024-05-01-preview) is the wiki entry for the features introduced in this service version. Maybe there is a bit more info we can share in the changelog.